builds/misc/templates/mqtt-perf-run.yaml

```diff
@@ -0,0 +1,265 @@
+jobs:
+
+  ################################################################################	
+  # Fan-in Test cases (many publishers/few subscribers)
+  ################################################################################	
+  
+  ### QoS 0
+  
+  - template: mqtt-perf-test-case.yaml
+    parameters: 
+      parentJob: setup_completed
+      id: 1
+      publishers: 100
+      subscribers: 2
+      topics: 1
+      payload: 32
+      qos: 0
+  
+  - template: mqtt-perf-test-case.yaml
```

Comment:
During setup stage, there are 2 devices set up for publisher and subscriber; and 1 agent set up to run mqtt broker.  Then there are many different test cases, and they will be run in sequence on the same publisher, subscriber, and mqtt broker without any setup/clean up between each test case.  Once all test cases completed, some clean up will be taken.

To confirm, we didn't do any clean up work at the beginning of test during setup stage, right? Would you tell me what clean up work will be done after all test cases ran?

Correct me if my understanding is incorrect.

Reply:
Philip, take a look at `builds/misc/templates/mqtt-perf-test-case.yaml`. At the end it does restart the container. This is the only cleanup we do b/w test cases, and it is enough. At the beginning of the pipeline (during setup stage) we do `docker system prune` as you recommended.

Reply:
is it better to remove the container between each test case run?

Reply:
Please create a PBI to track for this change.  thanks.

Reply:
It definitely would be better to re-create the container every time, but for that I need to propagate image name from artifact info down to the cleanup task, which would be cumbersome. I see what can be done, but will do that in a separate PR with other improvements to the perf tests.